server/routes/wazuh-utils/ui-logs.test.ts

```diff
@@ -0,0 +1,39 @@
+// To launch this file
+// yarn test:jest --testEnvironment node --verbose server/routes/wazuh-utils/ui-logs
+import axios from 'axios';
+
+function buildAxiosOptions(method: string, path: string, data: any = {}, headers: any = {}){
+  return {
+    method: method,
+    headers: { 'Content-Type': 'application/json', 'kbn-xsrf': 'kibana', ...headers },
+    url: `http://localhost:5601${path}`,
+    data: data
+  };
+};
+
+
+describe('Wazuh API - /utils/logs/ui', () => {
+  test('[200] Get UI Logs', () => {
+    const options = buildAxiosOptions('get', '/utils/logs/ui');
+    return axios(options).then(response => {
+      expect(response.status).toBe(200);
+      //expect(typeof response.data.data).toBe('object');
```

Comment:
is this commented code something we need? otherwise delete all code not being executed